### ISSUE TITLE
multiple tags ordering causes tag replacement

### ISSUE THREAD

**Issue by drewmullen** [hashicorp/terraform-provider-awscc#293]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment
* The resources and data sources in this provider are generated from the CloudFormation schema, so they can only support the actions that the underlying schema supports. For this reason submitted bugs should be limited to defects in the generation and runtime code of the provider. Customizing behavior of the resource, or noting a gap in behavior are not valid bugs and should be submitted as enhancements to AWS via the Cloudformation Open Coverage Roadmap.

### Affected Resource(s)

* any that accept tags, example:
* awscc_ec2_vpc

### Terraform Configuration Files

Please include all Terraform configurations required to reproduce the bug. Bug reports without a functional reproduction may be closed without investigation.

```hcl
resource "awscc_ec2_vpc" "test" {
    cidr_block = "10.0.0.0/16"
    tags = [{
      key = "test"
      value = "test"
    },
    {
      key = "test2"
      value = "test2"
    }]
}
```

### Expected Behavior

On 2nd `apply` no changes should be found

### Actual Behavior

changes are found due to tag ordering (even if they dont change in hcl)

hcl (hasnt changed)
```hcl
resource "awscc_ec2_vpc" "test" {
    cidr_block = "10.0.0.0/16"
    tags = [{
      key = "test"
      value = "test"
    },
    {
      key = "test2"
      value = "test2"
    }]
}
```

Plan output
```
$ tf plan
  # awscc_ec2_vpc.test will be updated in-place
  ~ resource "awscc_ec2_vpc" "test" {
      - enable_dns_hostnames    = false -> null
      - enable_dns_support      = true -> null
        id                      = "vpc-09dba01c04d2ed415"
      - instance_tenancy        = "default" -> null
      ~ tags                    = [
          ~ {
            ~ key   = "test2" -> "test"
            ~ value = "test2" -> "test"

            ~ key   = "test" -> "test2"
            ~ value = "test" -> "test2"
          },
        ]
        # (6 unchanged attributes hidden)
    }
```
### Steps to Reproduce

1. `terraform apply`
2. `terraform apply`

**Comment by ewbankkit** [comment]
@drewmullen Are you still seeing this with `v0.9.0`? I am currently unable to reproduce.

**Comment by drewmullen** [comment]
this was resolved with [0.9](https://github.com/hashicorp/terraform-provider-awscc/releases/tag/v0.9.0)! thanks, @ewbankkit !
